### PR TITLE
HiPSCat tool suite for large astronomical datasets (hipscat, hipscat-import, lsdb)

### PR DESCRIPTION
**Pull request by mjuric**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by mjuric** [comment]
Hi @conda-forge/staged-recipes, this is ready for review.

These are noarch packages, but Windows build fails as one of the dependencies is not available for windows (this is expected).

**Comment by mjuric** [comment]
CC: @conda-forge/help-python

File: recipes/hipscat/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+{% set version = "0.2.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/astronomy-commons/hipscat/archive/v{{ version }}.tar.gz
+  sha256: fe38197e62c49cfbf89115bf93a3b07ae99860bb24f4627458fde4d88a518172
+# curl -L https://github.com/astronomy-commons/hipscat/archive/v0.2.1.tar.gz | openssl sha256
```
Comment (pkgw) [inline recipes/hipscat/meta.yaml:11]: This commented curl line could be deleted.

**Review by pkgw — COMMENTED** [review]
Looks very nice, Mario. Do any of these packages have test suites that we could be running? The `pip check` and import tests cover very basic stuff but it would be nice to actually exercise the functionality in the testing stage.

File: recipes/hipscat/meta.yaml (hunk 1 above)
Comment (mjuric) [reply recipes/hipscat/meta.yaml]: Good catch, fixed.

**Comment by pkgw** [comment]
@mjuric It looks like you're working on adding test support, thanks! Since you're using Git versions of the packages here, I'm inferring that some upstream fixes are needed to get them working, so I can imagine that it might take a little while to get things fully working. If it would be valuable to you to have these packages in conda-forge sooner rather than later, I'd be happy to merge without detailed testing (using the latest formal releases rather than Git), and we can add the tests in the feedstocks later.

**Comment by mjuric** [comment]
Thanks @pkgw, but we managed to get the pytests added for all three packages (it was worth doing -- it'll make the releases more solid going forward).

Take look at the updated recipes when you get a chance & hope this looks ready for merging.

**Pull request merged**